konduit-serving-cli/src/main/java/ai/konduit/serving/cli/launcher/command/build/extension/ServeBuildCommand.java

```diff
@@ -0,0 +1,174 @@
+@Summary("Start a konduit server application")
+@Description("Start a konduit server application. " +
+        "The application is identified with an id that can be set using the `--serving-id` or `-id` option. " +
+        "The application can be stopped with the `stop` command. " +
+        "This command takes the `run` command parameters. To see the " +
+        "run command parameters, execute `run --help`\n\n" +
+        "Example usages:\n" +
+        "--------------\n" +
+        "- Starts a server in the foreground with an id of 'inf_server' using 'config.json' as configuration file:\n" +
+        "$ konduit serve -id inf_server -c config.json\n\n" +
+        "- Starts a server in the foreground with an id of 'inf_server' using 'config.json' as configuration file and CPU profile:\n" +
+        "$ konduit serve -id inf_server -c config.json -p CPU\n\n" +
+        "- Starts a server in the background with an id of 'inf_server' using 'config.yaml' as configuration file:\n" +
+        "$ konduit serve -id inf_server -c config.yaml -b\n" +
+        "--------------")
+@Slf4j
+public class ServeBuildCommand extends ServeCommand {
+
+    private String profileName;
+
+    @Option(shortName = "p", longName = "profileName", argName = "profile_name")
+    @Description("Name of the profile to be used with the server launch.")
+    public void setProfileName(String profileName) {
+        this.profileName = profileName;
+    }
+
+    @Override
+    public void run() {
+        File savePath = new File(DirectoryFetcher.getBuildDir(), String.format("%s-pipeline.json", getId()));
+        File mfJar = new File(DirectoryFetcher.getBuildDir(), String.format("%s-manifest.jar", getId()));
+
+        try {
+            JsonNode jsonConfiguration = getConfigurationFromFileOrString(configuration);
+            if (jsonConfiguration == null) {
+                out.format("Invalid configuration defined by: %s", configuration);
+                System.exit(1);
+            } else {
+                FileUtils.writeStringToFile(savePath, jsonConfiguration.get("pipeline").toString(), StandardCharsets.UTF_8);
+
+                Class<?> buildCliClass = Class.forName("ai.konduit.serving.build.cli.BuildCLI");
+                Object buildCli = buildCliClass.getConstructor().newInstance();
+                Method buildCliMainMethod = buildCliClass.getMethod("exec", String[].class);
+                buildCliMainMethod.setAccessible(true);
+
+                Profile profile = profileName != null ? ProfileCommand.getProfile(profileName) : ProfileCommand.getDefaultProfile();
+                if(profile == null) {
+                    if(profileName == null) {
+                        out.println("Couldn't find a default profile. Starting server with 'CPU' profile");
+                    } else {
+                        out.format("Couldn't find a profile with the specified name: '%s'. Starting server with 'CPU' profile.%n", profileName);
```

Comment:
We should probably just fail on this case instead (to avoid user being surprised by the behaviour after a profile name typo, for example)